src/solvers/kriging.jl

```diff
@@ -98,34 +99,30 @@ function preprocess(problem::EstimationProblem, solver::Kriging)
 
       # determine path and neighborhood search method
       if varparams.maxneighbors ≠ nothing
-        # locations with data for variable
-        varlocs = collect(keys(datamap(problem, var)))
-
         if varparams.neighborhood ≠ nothing
           # local search with a neighborhood
           neigh = varparams.neighborhood
 
-          # create a path from the data and outwards
-          # use at most 10^2 points to generate path
-          N = length(varlocs); M = ceil(Int, N/10^2)
-          path = SourcePath(view(varlocs,1:M:N))
-
-          searcher  = NeighborhoodSearcher(pdomain, neigh)
-          bsearcher = BoundedSearcher(searcher, maxneighbors)
+          if neigh isa BallNeighborhood
+            #for specialised knn and ball search combined
+            bsearcher = KBallSearcher(pdata, maxneighbors,neigh)
```

Comment:
Perhaps we can erase the comment, it is inferred from the code directly. Also, my pedantic mode asks for an extra space after the last comma, could you please fix these minor details?